Upgrade Test:Apicast dependency

eloycoto opened the pull request:
Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

eloycoto commented:
@roivaz is there a way to trigger circle here?

Regards

roivaz commented:
@eloycoto triggering the pipeline from forks is disabled. Is you push your branch directly in this repo, the pipeline should automatically trigger.

eloycoto committed:
Fix test:

- Add JUNIT to the package, main image changed.
- Fix LuaPath
- Use the correct image

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>
